src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,208 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int NULL = 0;
+    private static final int ONE = 1;
+    private Node head;
+    private Node tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        add(value, size);
     }
 
     @Override
     public void add(T value, int index) {
+        validateIndexInclusive(index);
```

Comment:
Try to implement it this way:
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/3ed57c9b-a561-41e0-b470-e2c76965d599)